hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -35,12 +48,52 @@ Check that the output `0x3020000a` is shown. If it is not, then the OTP bit has
 
 If you wish, you can remove the `program_usb_boot_mode` line from config.txt, so that if you put the SD card into another Raspberry Pi, it won't program USB host boot mode. Make sure there is no blank line at the end of config.txt.
 
-## Prepare the USB mass storage device
+You can now boot from an USB Mass Storage device in the same way as booting from SD, see the following paragraph for Raspberry Pi 3B+.
+
+## Raspberry Pi 3B+
+
+The Raspberry Pi 3B+ supports USB Mass Storage boot out of the box. The settings specific to the previous versions of Raspberry Pi do not have to be executed.
+
+This is verified using the following steps, this is the same (procedure)[https://www.raspberrypi.org/documentation/installation/installing-images/] as for SD cards though many alternative ways are functional:
+
+1. Download Raspbian Buster Lite
+
+2. Unzip the .zip file (tested with 7zip, but any other should be fine) to an .img file.
+
+3. Initialise the drive (tested with MBR scheme, no partition/clean disk).
```

Comment:
This step is never necessary, as flashing the .img overwrites the MBR

Reply:
It actually was required on Windows for my 'clean' harddrive to get assigned a drive letter. Before I was able to write to it using Win32DiskImager.

In fact; I'm not even sure if we should include these steps here, as it will lead to double documentation and it has two issues:
- If you update one; you would have to update the other.
- There's two places with the same info; so it may be harder to find in search results.
- It leads to confusion as to which is the "correct" solution.

Reply:
:+1: for not having duplicated documentation.